src/solvers/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ if("${sat_impl}" STREQUAL "minisat2")
         URL http://ftp.debian.org/debian/pool/main/m/minisat2/minisat2_2.2.1.orig.tar.gz
         PATCH_COMMAND patch -p1 -i ${CBMC_SOURCE_DIR}/../scripts/minisat-2.2.1-patch
         COMMAND cmake -E copy ${CBMC_SOURCE_DIR}/../scripts/minisat2_CMakeLists.txt CMakeLists.txt
+        URL_MD5 27faa19ee0508660bd6fb7f894646d42
```

Comment:
That's a very nice addition! But then: is this really the only place where this is to be done? At least Glucose a few lines further down should see the same.

Reply:
done